.changeset/hip-buttons-dance.md

```diff
@@ -0,0 +1,5 @@
+---
+"@blitzjs/generator": patch
+---
+
+Remove random user git config when generating a new project & commiting.
```

Comment:
Hmm, we could be more clear about what changes for the users by saying that now we're going to use users' git account to commit instead of a `Blitz.js CLI` (`noop@blitzjs.com`).